generators/app/files.js

```diff
@@ -126,6 +127,7 @@ module.exports.API_CONFIG = `${module.exports.CONFIG_PATH}/api.js`;
 module.exports.CONFIG = `${module.exports.CONFIG_PATH}/index.js`;
 module.exports.REACTOTRON_CONFIG = `${module.exports.CONFIG_PATH}/ReactotronConfig.js`;
```

Comment:
```
// TODO: Normalize config filenames
```

Reply:
Agree. I'd create a card to do this later.